modules/igv/1.0/config/default.yaml

```diff
@@ -0,0 +1,118 @@
+lcr-modules:
+    
+    igv:
+
+        inputs:
+            # Available wildcards: {unix_group} {seq_type} {tumour_sample_id} {normal_sample_id} {pair_status} {genome_build}
+            maf: "__UPDATE__"
+
+        regions: 
+            # Provide regions files as lists in their respective genome builds so that liftover of coordinates occurs properly
+            oncodriveclustl:
+                grch37: []
+                hg38: []
+            hotmaps:
+                grch37: []
+                hg38: []
+            bed:
+                grch37: []
+                hg38: []
+            maf:
+                grch37: []
+                hg38: []
+            mutation_id:
+                grch37: []
+                hg38: []
+
+        # Stop snakefile after MAF filtering step to estimate total number of snapshots that will be taken without running IGV
+        estimate_only: False 
+
+        options:
+            genome_map: 
+                # Map metadata builds to grch37 and hg38 so that MAF file locations are determined correctly
+                grch37: ["__UPDATE__"] # e.g ["grch37","hg19","hs37d5"]
+                hg38: ["__UPDATE__"] # e.g ["hg38","grch38"]
+
+            liftover_regions:
+                reference_chain_file:
+                    grch37: "genomes/grch37/chains/grch37/hg19ToHg38.over.chain"
```

Comment:
I think this is generated with the reference files so you can directly request the reference files output instead of having this listed here. This can be an example:
https://github.com/LCR-BCCRC/lcr-modules/blob/7b495a3e92c3813cc58ff6569c92476cbc6e53c0/modules/liftover/2.0/liftover.smk#L108-L130

Reply:
Done ✅